### DISCUSSION TITLE
New escape flag for completions

### DISCUSSION THREAD

**Discussion by pikeas** [PatrickF1/fzf.fish#222] [Ideas]
This feature just landed in fish shell and I thought it might be helpful for your (wonderful!) plugin: https://github.com/fish-shell/fish-shell/pull/8645

**Comment by PatrickF1** [comment]
Thanks for the idea! The next version of fish isn't released yet but I'd love to get ahead on the updates. Since our completions don't include any special characters, why do you think I should include the new flag?

Btw, when this new completions feature does GA, feel free to open a PR to include it!
